pages/verify-identity.tsx

```diff
@@ -67,16 +146,24 @@ export default function IdentityVerificationForm() {
             20th August 1950, you would enter 20-08-1950`}
               </FormHelperText>
             </FormControl>
+            {data?.verifyIdentity.failureReason && (
+              <Alert status="error" marginBottom="20px">
+                <AlertIcon />
+                <AlertDescription>
+                  {getErrorMessage(data.verifyIdentity.failureReason)}
+                </AlertDescription>
+              </Alert>
+            )}
             <Flex width="100%" justifyContent="flex-end">
               <Link href="/">
                 <Button variant="outline" marginRight="12px">{`Go back to home page`}</Button>
               </Link>
-              <Link href="/renew">
-                <Button
-                  variant="solid"
-                  disabled={!userId || !phoneNumberSuffix || !dateOfBirth}
-                >{`Continue`}</Button>
-              </Link>
+              <Button
+                variant="solid"
+                disabled={loading || !userId || !phoneNumberSuffix || !dateOfBirth}
```

Comment:
Do we need loading on this as well?

Reply:
Yup, we need to call the API to verify the user's identity